common/scala/src/main/scala/org/apache/openwhisk/core/containerpool/Container.scala

```diff
@@ -204,6 +209,17 @@ trait Container {
         RunResult(Interval(started, finished), response)
       }
   }
+  private def openConnections(timeout: FiniteDuration, maxConcurrent: Int) = {
+    if (Container.config.akkaClient) {
```

Comment:
I forgot what we're still maintaining the Akka Client and the Apache Client, since the latter was found in https://github.com/apache/incubator-openwhisk/pull/3812 to perform poorly.

Reply:
are we ready to move to the akka client exclusively? there were some reservations about making sure we didn't get bit by some akka issues as in the past.

Reply:
I'm ready 😄 
FWIW, we have been using akka client exclusively in prod, with concurrency enabled, since December.
Tests are already using it as well. 
I'm not sure if there are others that are interested in testing it more, but it would be great to retire the apache client IMHO.

Reply:
Let's confirm that via dev-list.